corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java

```diff
@@ -72,6 +76,9 @@ public class CorfuStoreBrowserEditor {
     private final DynamicProtobufSerializer dynamicProtobufSerializer;
     private final String QUOTE = "\"";
 
+    @Setter
+    private int maxRecordsInBatch = 100000;
```

Comment:
Since this is used only in deleteRecords, could this be passed as an argument instead of making it a class member?

Reply:
Done. thanks